website/google.erb

```diff
@@ -46,23 +46,23 @@
       <li>
         <a href="#">Resources</a>
         <ul class="nav nav-auto-expand">
-  
+
```

Comment:
Can you revert the changes to this file? Looks like they are spacing changes and shouldn't matter